### ISSUE TITLE
Fine_tune.run_fine_tune returns losses of training set rather than validation set

### ISSUE THREAD

**Issue by ShinnosukeUesaka** [mukobi/Unadaptable-Foundation-Models#32]
Fine_tune.run_fine_tune should return the losses of validation set to calculate LOSS GAP RATIO. The current implementation returns losses of training set.

**Comment by owen-yeung** [comment]
Hmm I realize you wrote this before I wrote run_fine_tune
Was someone else working on this?

**Comment by ShinnosukeUesaka** [comment]
I was writing the skeleton of run_fine_tune in #23, but we can replace it with your code
